pkg/test/runner/runner.go

```diff
@@ -51,19 +65,26 @@ const (
 )
 
 // NewRunner returns a new runner for pkg
-func NewRunner(t *testing.T, testCase TestCase, c string) (*Runner, error) {
+func NewRunner(t *testing.T, testCase TestCase, c, kptBin string) (*Runner, error) {
 	info, err := os.Stat(testCase.Path)
 	if err != nil {
 		return nil, fmt.Errorf("cannot open path %s: %w", testCase.Path, err)
 	}
 	if !info.IsDir() {
 		return nil, fmt.Errorf("path %s is not a directory", testCase.Path)
 	}
+	if kptBin == "" {
```

Comment:
Can this be optional ? I think current way of running test is also useful.

Is the kpt developer expected to build kpt first and then run the test might lead to human errors where test were invoked without building the kpt binary.